### PR TITLE
Bump minimist to 1.2.5

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Manually bumps minimist to 1.2.5 to deal with CVE-2020-7598

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by balloob — DISMISSED** [review]

**Comment by balloob** [comment]
The problem with this kind of fix and the lack of comments, is that we cant know when to drop it in the future. 

Thinking of it now, better to use the resolutions field

**Comment by frenck** [comment]
Good call, changed it. Lock remained the same with that change.

File: package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "textlint-rule-common-misspellings": "^1.0.1",
     "textlint-rule-terminology": "^2.1.4"
   },
+  "resolutions": {
+    "minimist": "^1.2.5"
```
Comment (balloob) [inline package.json]: So I'm not sure if npm supports it but can we do >= so we don't pin any version, just set constraints?

Comment (frenck) [reply package.json]: yes it does. Updated.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**